addons/account/company.py

```diff
@@ -34,6 +34,20 @@ class ResCompany(models.Model):
     property_stock_account_input_categ = fields.Many2one('account.account')
     property_stock_account_output_categ = fields.Many2one('account.account')
     property_stock_valuation_account_id = fields.Many2one('account.account')
+    bank_journal_ids = fields.One2many('account.journal', 'company_id', domain=[('type', '=', 'bank'), ('bank_account_id', '!=', False)],
```

Comment:
I think we need this field in order to display the company's bank accounts on the company form view. If I'm not mistaken we have to choose to either keep this field or (cleaner but maybe less user-friendly) to company's bank accounts must be configured in accounting > config > journals (we could add a link 'configure bank accounts' in the company's form view)